### DISCUSSION TITLE
Android estudio não funcionado ctrl + M

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1750] [Perguntas e Respostas]
Boa noite devs!

Estou estudando React Native, porem estou enfretando um problema no Android estudio
não consigo mudar de pagina no emulador e o atalho que éra para ir para o menu "ctrl + M" 
não está funcionando.

Fiz umas pesquisas antes,  e diz em alguns sites que seria umas variaveis que teria
que adicionar no windows , alguem já passou por este problema ou sabe como resolver?

**Comment by akinncar** [comment]
Qual emulador você está utilizando? e qual a versão do react native do projeto? se o emulador também tiver a opção "Shake device" para simular o dispositivo sendo chacoalhado e ativar o dev menu, você pode usa-la pra ver se o menu aparece, caso aparecer saberemos que o problema está em mapear o cntrl + M, e não no menu
